specs/merge/beacon-chain.md

```diff
@@ -240,19 +241,31 @@ def compute_timestamp_at_slot(state: BeaconState, slot: Slot) -> uint64:
 The implementation-dependent `ExecutionEngine` protocol encapsulates the execution sub-system logic via:
 
 * a state object `self.execution_state` of type `ExecutionState`
-* a state transition function `self.on_payload` which mutates `self.execution_state`
+* a state transition function `self.execute_payload` which applies changes to the `self.execution_state`
+* a function `self.consensus_validated` which signals that the beacon block containing the execution payload
```

Comment:
Nitpicking on the function name conventions. Active voice v.s. passive voice.

I saw there is `engine_consensusValidated` in Engine APIs, but this naming pattern seems new to consensus-specs. How about renaming it to `call_consensus_validated`?

Also suggesting `forkchoice_updated` -> `call_forkchoice_updated`.


ref. https://developers.google.com/style/voice

Reply:
`engine_consensusValidated` and `engine_forkchoiceUpdated` are rather notifications than methods, we may use the `notify_` or `signal_` prefix for names of functions sending notifications. WDYT?

Reply:
I prefer `notify_` or `signal_` if we want to move it to the active voice.

Reply:
Do you mean also changing the Engine APIs specs?

`notify_` and `signal_` both sound good to me. Maybe `notify_` is more common? 🙂

Reply:
probably not change the engine api specs, imo. but making this call still map to the api specs but conform a bit more to our verb usage is good

Reply:
Yeah, just renaming it here in the beacon spec code. Execution engine is an abstraction in the beacon spec and Engine API client is one of potential implementations of this abstraction. I'll do the renaming

Reply:
Done